apps/earn-protocol/app/api/forecast/[...forecastParams]/route.ts

```
@@ -0,0 +1,183 @@
+    return NextResponse.json({ error: 'Invalid forecast params' }, { status: 400 })
+  }
+
+  if (!isValidAddress(fleetAddress)) {
+    return NextResponse.json({ error: 'Invalid fleet address' }, { status: 400 })
+  }
+
+  if (!isSupportedSDKChain(Number(chainId))) {
+    return NextResponse.json({ error: 'Invalid chain ID' }, { status: 400 })
+  }
+
+  if (isNaN(Number(amount)) || Number(amount) <= 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)